alert.css

```diff
@@ -0,0 +1,28 @@
+.alert-container{
+ 
+    display : flex;
+    padding : 1.25rem;
+    font-size: 1.25rem;
+    font-family: "comic sans ms";
+    width : 50vw;
+    border-radius : 0.625rem;
+    justify-content: space-between;
+    margin : 1.25rem;
+}
+
+.alert-container-error{
+    background-color: rgb(245, 125, 125);
```

Comment:
It will be good if you pre-define these colors in root and import them as var().

Reply:
done! Thanks for the review